php/basicphp/Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM php:7.3-fpm
```

Comment:
@ray-ang You can remove this file :stuck_out_tongue: 

I have made `Dockerfile` creation dynamic using https://github.com/the-benchmarker/web-frameworks/blob/master/php/Dockerfile and https://github.com/the-benchmarker/web-frameworks/blob/master/php/basicphp/config.yaml

Reply:
Removed Dockerfile as advised.